templates/kubernetes/features.html

```diff
@@ -0,0 +1,160 @@
+{% extends "kubernetes/base_kubernetes.html" %}
+{% block title %}Features of Canonical Kubernetes on Ubuntu{% endblock %}
+{% block meta_description %}Canonical OpenStack on Ubuntu includes software-defined networking and storage options, architectural flexibility, and shared community-driven ops code independent of architecture.{% endblock meta_description %}
+
+{% block content %}
+
+<section class="p-strip is-deep is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h1>Canonical Distribution of Kubernetes features</h1>
+      <p>Architectural freedom. Fully automated operations. Extensible Kubernetes for all.</p>
```

Comment:
I think this one should be a `p-heading--four`

Reply:
Done